### DISCUSSION TITLE
Specify alternative model source for SceneViewer only?

### DISCUSSION THREAD

**Discussion by milesgreen** [google/model-viewer#2339] [Q&A]
Is it possible to specify a different GLB source for just SceneViewer to use rather than the ModelViewer `src` itself?

Similar to how we specify an alternative `ios-src` for Apple AR QuickLook. 

Reason being, ModelViewer (and WebXR) now support a range of extended functionality with models - material variants, animation controls, hotspots, orientation/scale manipulation - enabling pretty interactive experiences. These features are not supported by AR QuickLook or SceneViewer. It would be useful to be able to set a 'simplified' GLB that can be specified for SceneViewer only that can take into account the lack of these features. 

Another use case might be for Material Variants. As the user switches between variants, the `ios-src` and SceneViewer source properties could be updated through code to point to a pre-baked model of just that variant, so when switching to AR, Quick Look and SceneViewer can display that variation.

**Comment by david-rhodes** [comment]
This would be a great addition!

**Reply by elalish** [reply]
I think we're more likely to head in the direction of bringing the Scene Viewer features to WebXR in order to avoid this bifurcation. We would also like to remove the need for `ios-src` through runtime USDZ conversion.

**Reply by milesgreen** [reply]
OK. Makes sense. In the meantime, I think I can work around it.
Runtime USDZ conversion would certainly be epic, especially if worked in conjunction with Material Variants. 
Or, again, if Apple could just wake up on the whole WebXR thing...
